tests/api/test_bridge.py

```diff
@@ -0,0 +1,26 @@
+from pprint import pprint
+
+
+class TestSaiBridge:
+    # object with no parents
+
+    def test_bridge_create(self, npu):
+        commands = [
+            {
+                'name': 'bridge_1',
+                'op': 'create',
+                'type': 'SAI_OBJECT_TYPE_BRIDGE',
+                'attributes': ['SAI_BRIDGE_ATTR_TYPE', 'SAI_BRIDGE_TYPE_1Q'],
```

Comment:
From what I recall, only default .1q bridge is supported by SAI spec. So, this operation is expected to fail.

Reply:
That's because CI/CD uses saivs which is not 100% SAI spec compliant. In fact, it's rather stubs then real implementation.

Reply:
i am not sure what to be done for this case, should i remove this case ?

Reply:
I'd recommend to create `SAI_BRIDGE_TYPE_1D` instead + probably check that `SAI_SWITCH_ATTR_DEFAULT_1Q_BRIDGE_ID` is not `oid:0x0`

Reply:
removing this file